src/sentry/workflow_engine/typings/notification_action.py

```diff
@@ -412,3 +459,18 @@ class AzureDevOpsDataBlob(TicketDataBlob):
 
     project: str = ""
     work_item_type: str = ""
+
+
+@dataclass
+class SentryAppFormConfigDataBlob(DataBlob):
+    """Represents a single form config field for a Sentry App."""
```

Comment:
nit: doc comments are usually separated with newlines:
```suggestion
    """
    Represents a single form config field for a Sentry App.
    """
```